package.json

```diff
@@ -1,40 +1,42 @@
 {
-  "name": "malker",
-  "private": true,
-  "version": "0.0.1",
-  "type": "module",
-  "scripts": {
-    "dev": "vite",
-    "build": "vite build",
-    "lint": "npx oxlint && npx eslint",
-    "preview": "vite preview",
-    "start-api": "cd backend && venv/bin/flask run --no-debugger",
-    "format": "prettier --write 'src/**/*.{js,jsx}'"
-  },
-  "dependencies": {
-    "axios": "^1.6.8",
-    "prettier-plugin-tailwindcss": "^0.5.12",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-icons": "^5.0.1",
-    "react-loader-spinner": "^6.1.6",
-    "react-router-dom": "^6.21.1",
-    "react-toastify": "^9.1.3"
-  },
-  "devDependencies": {
-    "@types/react": "^18.2.15",
-    "@types/react-dom": "^18.2.7",
-    "@vitejs/plugin-react": "^4.0.3",
-    "autoprefixer": "^10.4.16",
-    "eslint": "^8.45.0",
-    "eslint-plugin-oxlint": "^0.2.3",
-    "eslint-plugin-react": "^7.32.2",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-react-refresh": "^0.4.3",
-    "oxlint": "^0.1.2",
-    "postcss": "^8.4.31",
-    "prettier": "^3.2.5",
-    "tailwindcss": "^3.3.5",
-    "vite": "^4.5.2"
-  }
+    "name": "malker",
+    "private": true,
+    "version": "0.0.1",
+    "type": "module",
+    "scripts": {
+        "dev": "vite",
+        "build": "vite build",
+        "lint": "npx oxlint && npx eslint",
+        "preview": "vite preview",
+        "start-api": "cd backend && venv/bin/flask run --no-debugger",
+        "format": "prettier --write 'src/**/*.{js,jsx}'",
+        "cypress": "npx cypress run --spec cypress/e2e/app.cy.js"
```

Comment:
Except this one. Disable your prettier extension for this files... Only JS and JSX